### DISCUSSION TITLE
Is it possible to hide events from an specific application?

### DISCUSSION THREAD

**Discussion by GroosL** [evilsocket/opensnitch#655] [General]
I have an IPFS node running but in just a few minutes is made +60k requests and it's impossible to check any other events because of this. Is it possible to hide any events related to this application?

**Comment by NRGLine4Sec** [comment]
Currently I think it's impossible, but I agree it would be a nice feature to have.

**Reply by gustavo-iniguez-goya** [reply]
yes, not possible right now.
